content/en/_index.html

```diff
@@ -8,7 +8,7 @@
 
 {{< blocks/section id="oceanNodes" >}}
 {{% blocks/feature image="flower" %}}
-[Kubernetes]({{< relref "/docs/concepts/overview/" >}}), also known as K8s, is an open-source system for automating deployment, scaling, and management of containerized applications.
+[Kubernetes]({{< relref "/content/en/releases/download.md" >}}), also known as K8s, is an open-source system for automating deployment, scaling, and management of containerized applications.
```

Comment:
This isn't right I'm afraid. Issue https://github.com/kubernetes/website/issues/28054 is about adding something _new_ to the page at https://k8s.io/ whereas this change would amend what's already there.

How about putting the link in a new sentence under the Run K8s Anywhere heading?